conda_smithy/configure_feedstock.py

```diff
@@ -1766,6 +1767,7 @@ def get_common_scripts(forge_dir):
         "run_docker_build.sh",
         "build_steps.sh",
         "run_osx_build.sh",
+        "utils.sh",
```

Comment:
```suggestion
        "logging_utils.sh",
```

Reply:
Oops, actually we don't need this anymore, since they are static. I have to remove it. Thanks!